src/pip/_internal/download.py

```diff
@@ -692,55 +692,57 @@ def is_secure_origin(self, location):
         # Determine if this url used a secure transport mechanism
         parsed = urllib_parse.urlparse(str(location))
         origin = (parsed.scheme, parsed.hostname, parsed.port)
```

Comment:
Since we don't use it anymore, can we remove `origin` and just assign to `origin_protocol`, `origin_host`, and `origin_port` directly?

Reply:
Sure. I originally had it that way, but then I broke it up to fit on one line. Happy to change it back.